recipes-core/images/aos-image-vm.bb

```diff
@@ -44,6 +44,7 @@ IMAGE_INSTALL_append = " \
 # Variables
 
 BUILD_WIC_DIR="${WORKDIR}/build-wic"
+ROOTFS_EXCLUDE_PATHS = "${@bb.utils.contains('DISTRO_FEATURES', 'selinux', '', '--exclude-path ./var/ ./home/', d)}"
```

Comment:
Could you add comment here, why we need to remove exclude path for SE linux? In one sentence describe labeling issue.

Reply:
Sure, I've added small comment with description